grails-app/domain/org/pih/warehouse/order/Order.groovy

```diff
@@ -286,7 +286,11 @@ class Order implements Serializable {
     }
 
     def getShipments() {
```

Comment:
can you add a brief comment in the code here as to why we need to filter out nulls? It's a complex enough situation to not immediately be obvious.